test/CMakeLists.txt

```diff
@@ -85,13 +85,14 @@ include( ${MODULE_ROOT_DIR}/tools/cmock/create_test.cmake )
 
 # Include build configuration for unit tests.
 add_subdirectory( unit-test )
+add_subdirectory( unit-test-http )
```

Comment:
Why not just put it under `unit-test`?

Reply:
For data over HTTP we need to set the configuration in ota_config.h
We would need a separate folder to keep this file.